src/components/ElevationSlider.vue

```diff
@@ -35,68 +35,61 @@ function roundToNearest100 (x: number) {
   }
 })
 export default class ElevationSlider extends Vue {
-  @Prop({ default: 0 }) value!: number
+  @Prop({ default: 1 }) value!: number
   @Prop({ default: 0 }) minValue!: number
-  @Prop({ default: 0 }) maxValue!: number
+  @Prop({ default: 10 }) maxValue!: number
 
   currentValue: number = 0
   readonly numberOfMarks: number = 8
   marks: number[] = []
 
-  mounted() {
+  beforeMount() {
     this.onValueChange()
```

Comment:
The onValueChange on the beforeMount may give issues if we later will add logic to the onValueChange routine. Therefore setting a default value is better to do inline I think